torchani/ase.py

```diff
@@ -46,8 +46,7 @@ def __init__(self, species, aev_computer, model, energy_shifter, dtype=torch.flo
         self.device = self.aev_computer.EtaR.device
         self.dtype = dtype
 
-        self.whole = torch.nn.Sequential(
-            self.aev_computer,
```

Comment:
You are missing a `self.aev_computer.to(self.dtype)`

Reply:
Resolved in 92584cc